ONSdigital/rm-collection-exercise-service
# spring boot version upgrade

# What and why?
Upgrading spring boot to 2.6.6 like the other Java services.

# How to test?

- Deploy fresh copy of your environment
- Delete the `collectionexercise` schema
- Deploy this PR to your environment. It should deploy without any errors.
- Run acceptance tests, they should pass
- Go on R-ops and create a collection exercise/set it to live.
- `kc rollout restart deployment collection-exercise` to restart it, the data should remain.

# Trello
[Card](https://trello.com/c/IJWjGKz1)
